tests/behaviour/qbwriter.feature

```diff
@@ -203,207 +203,208 @@ Feature: Test outputting CSV-Ws with Qb flavouring.
     When the cube is serialised to CSV-W
     Then csvlint validation of "single-measure-qube-with-existing-resources.csv-metadata.json" should succeed
     And csv2rdf on all CSV-Ws should succeed
     And some additional turtle is appended to the resulting RDF
-    """
+      """
       @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
       @prefix qb: <http://purl.org/linked-data/cube#>.
       @prefix skos: <http://www.w3.org/2004/02/skos/core#>.
       @prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
 
       <http://existing/dimension>
-          rdfs:label "Some Existing Dimension"@en;
-          a qb:DimensionProperty;
-          qb:codeList <http://existing/dimension/code-list>;
-          rdfs:range <http://some/range/thingy>.
+      rdfs:label "Some Existing Dimension"@en;
+      a qb:DimensionProperty;
+      qb:codeList <http://existing/dimension/code-list>;
+      rdfs:range <http://some/range/thingy>.
 
       <http://existing/dimension/code-list> a skos:ConceptScheme;
-        skos:hasTopConcept <http://existing/dimension/code-list/all>.
+      skos:hasTopConcept <http://existing/dimension/code-list/all>.
 
       <http://existing/dimension/code-list/all> a skos:Concept;
-        rdfs:label "All possible things"@en;
-        skos:inScheme <http://existing/dimension/code-list>.
+      rdfs:label "All possible things"@en;
+      skos:inScheme <http://existing/dimension/code-list>.
 
       <http://existing/dimension/code-list/a> a skos:Concept;
-        rdfs:label "A"@en;
-        skos:inScheme <http://existing/dimension/code-list>;
-        skos:broader <http://existing/dimension/code-list/all>.
+      rdfs:label "A"@en;
+      skos:inScheme <http://existing/dimension/code-list>;
+      skos:broader <http://existing/dimension/code-list/all>.
 
       <http://existing/dimension/code-list/b> a skos:Concept;
-        rdfs:label "B"@en;
-        skos:inScheme <http://existing/dimension/code-list>;
-        skos:broader <http://existing/dimension/code-list/all>.
+      rdfs:label "B"@en;
+      skos:inScheme <http://existing/dimension/code-list>;
+      skos:broader <http://existing/dimension/code-list/all>.
 
       <http://existing/dimension/code-list/c> a skos:Concept;
-        rdfs:label "C"@en;
-        skos:inScheme <http://existing/dimension/code-list>;
-        skos:broader <http://existing/dimension/code-list/all>.
+      rdfs:label "C"@en;
+      skos:inScheme <http://existing/dimension/code-list>;
+      skos:broader <http://existing/dimension/code-list/all>.
 
       <http://existing/attribute> a qb:AttributeProperty;
-          rdfs:label "Some existing attribute property"@en.
+      rdfs:label "Some existing attribute property"@en.
 
       <http://existing/measure> a qb:MeasureProperty;
-          rdfs:label "Some existing measure property"@en;
-          rdfs:range xsd:decimal.
+      rdfs:label "Some existing measure property"@en;
```

Comment:
Can you undo all of these indentation changes? They make it harder to read the RDF.